src/main/java/edu/unc/lib/boxc/migration/cdm/model/CdmFieldInfo.java

```diff
@@ -16,14 +16,27 @@
 package edu.unc.lib.boxc.migration.cdm.model;
 
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.jena.ext.com.google.common.collect.Streams;
 
 /**
  * Information describing the fields present in a CDM collection
  *
  * @author bbpennel
  */
 public class CdmFieldInfo {
+    public static final String CDM_ID = "cdmid";
```

Comment:
Should this be its own constant if no other field is?

Reply:
I made a this one into a constant because I needed to reference the field in another location down here https://github.com/UNC-Libraries/cdm-to-boxc-migration-util/pull/3/files/c22414f4f71187771a869e4762ccfece64ac086b
I didn't need to refer to any of the other reserved fields by name anywhere else so far, but i would probably make constants for them if i end up needing to do so